types/node/events.d.ts

```diff
@@ -35,9 +35,6 @@ declare module "events" {
          */
         let captureRejections: boolean;
 
-        interface EventEmitter extends NodeJS.EventEmitter {
-        }
-
         class EventEmitter {
```

Comment:
For backwards compatibility, `'events'.EventEmitter` should keep extending `NodeJS.EventEmitter`:
```suggestion
        interface EventEmitter extends NodeJS.EventEmitter {
        }

        class EventEmitter {
```

Reply:
This should not matter as they still result in the same effective type definition.
I haven't found *any* packages that had issues with the reversal in inheritance (hence no NodeJS.EventEmitter related changes in other packages).

Reply:
That said if you can point to a potential issue I'm willing to change my mind.